tests/integerationTests/bookmarksContainer.test.jsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import { shallow } from "enzyme";
+
+import {
+  Bookmarks,
+  CardLayout
+} from "../../src/containers/bookmarks/Bookmarks";
+
+describe("When the bookmarks component is mounted", () => {
+  describe("<Bookmarks />", () => {
+    test("renders the component", () => {
```

Comment:
could you be consistent to either use test or it in the test case

Reply:
Removed the inconsistence in the tests